libs/blocks/section-metadata/section-metadata.css

```diff
@@ -62,3 +62,7 @@
 .section.heading-center h6 {
   text-align: center;
 }
+
+.section.text-center p {
+  text-align: center;
+}
```

Comment:
Can we simplify this whole thing? Why does an author need to spec text-center and heading-center?

Also, seems like we should only affect direct descendants. I worry that this is too pervasive and will affect blocks who don't spec a text-aling: left property.

Reply:
I addressed the second item.

@fullcolorcoder It looks like you originally added `.heading-center`. Do you know if it'd be ok to combine these or is there a use case for them being separate?

Reply:
@meganthecoder It won't impact anything already authored to combine them.

Reply:
@fullcolorcoder Great. Does that mean `heading center` isn't being used yet? Because if we combine them, it doesn't really make sense to call it that since it's more than headings. Just `center` or `text center` makes more sense to me. @auniverseaway Do you have a preference?

Reply:
Center is great.

FWIW: I always try to do `verb noun` on Helix as it sounds more human-centric. 

In my mind, a human would likely say, "Center text." instead of "Text, center."

Reply:
@auniverseaway Updated. Please take another look